src/functions-and-arrays.js

```diff
@@ -1,20 +1,50 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(numb1,numb2) {
+
+  if (numb1 > numb2){
+return numb1;
+}
+else if(numb1 < numb2){
+  return numb2;
+} else if (numb1 === numb2){
+  return numb1 || numb2;
+}
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  if(arr.length === 0){
+    return null;
+  }else if (arr.length === 1){
+      return arr[0];
+    }
+Math.max(...(x.map(el => el.length)));
+
+// for (let i = 0; i<)
+}
 
 
 
```

Comment:
for iteration 2 I'm not sure if you understood the task correctly.
One way to solve the task would have been like this:
```
function findLongestWord(arr) {
  let longestWord = "";

  if (arr.length === 0) {
    return null;
  }
  for (let i = 0; i < arr.length; i++) {
    if (arr[i].length > longestWord.length) {
      longestWord = arr[i];
    }
  }

  return longestWord
}
```